CMakeLists.txt

```diff
@@ -611,14 +611,16 @@ else (UNIX)
       set(sdk_bindir "${sdk_dir}/bin")
     endif (X64)
   endif ()
-  find_program(CMAKE_MC_COMPILER mc.exe HINTS "${sdk_bindir}"
-    DOC "path to message compiler")
-  if (NOT CMAKE_MC_COMPILER)
-    message(FATAL_ERROR "message compiler not found: required to build")
-  endif (NOT CMAKE_MC_COMPILER)
-  message(STATUS "Found message compiler: ${CMAKE_MC_COMPILER}")
-  mark_as_advanced(CMAKE_MC_COMPILER)
-
+  if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio")
```

Comment:
The Ninja generator also uses the Visual Studio compiler, so checking the generator is not sufficient.  What is needed is something like what we have for Unix, CMAKE_COMPILER_IS_CLANG: we'd want to add CMAKE_COMPILER_IS_VISUAL_STUDIO or something.

Reply:
Fixed.